client/public/index.html

```diff
@@ -5,9 +5,10 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+    <link rel="preload" href="<%= BASE_URL %>fonts/materialdesignicons-webfont.927457ed.woff2" as="font" type="font/woff2" crossorigin>
```

Comment:
Did you do anything to remove the MaterialIcons (not MDI) that are copied to the dist folder?

Reply:
No, though i'm not entirely sure what you mean.

Reply:
Hmm, looks like importing the Material Icons CSS is enough to load them. It also looks like the Material Design Icons CSS does the same thing. So maybe this line isn't needed here???

There is something about preloading fonts so the app has them available right away, but this doesn't seem to be needed for icons.

Do the MDI still load without this line?

Reply:
I tested locally in Filings UI and it appears that this line is NOT required.

Reply:
Copy that, removed in next commit!